src/Components/NavBar/MinimalNavBar.tsx

```diff
@@ -21,7 +21,7 @@ export const MinimalNavBar: React.FC<MinimalNavBarProps> = props => {
     >
       <AppContainer>
         <Box height={70} px={[2, 4]}>
-          <RouterLink to={props.to}>
+          <RouterLink to={props.to} className="acceptance__logoLink">
```

Comment:
@damassi can you please use the `data-test` attribute here as we do in [other places](https://github.com/artsy/reaction/pull/3089)?

It's a small change but best to be consistent!

Reply:
Oh this is already in :)

Reply:
I know! I am suggesting a quick update.

The integrity tests are primarily failing for another reason (an inability to access cms-staging), so there's not a huge rush over there. 😄

Reply:
(No meaning, i've already updated reaction with `data-test`)

Reply:
Oooh 👍